slice/IceLocatorDiscovery/IceLocatorDiscovery.ice

```diff
@@ -5,14 +5,12 @@
 [["cpp:doxygen:include:IceLocatorDiscovery/IceLocatorDiscovery.h"]]
 [["cpp:header-ext:h"]]
 
+[["java:package:com.zeroc"]]
 [["js:module:@zeroc/ice"]]
-
 [["python:pkgdir:IceLocatorDiscovery"]]
 
 #include "Ice/Locator.ice"
 
-[["java:package:com.zeroc"]]
```

Comment:
The old ordering was odd - why did we put `[[java:package:...]]` after the include?